### PR TITLE
Feature/add telemetry

### PR DESCRIPTION
**Pull request by Kam0ss**
Telemetry added in AUV8 and local compose.

### CODE REVIEW

File: docker-compose.local.yml [hunk 1]
```diff
@@ -1,181 +1,184 @@
...
       network_mode: host
-      environment:
-        - ROS_MASTER_URI=http://${ADRESS_IP}:11311
-        - ROS_IP=${ADRESS_IP}
-      ports:
-        - "9090:9090"
       depends_on:
         - ros-master
+      environment: 
+        - ADRESS=
```
Comment (0x72D0) [inline docker-compose.local.yml]: I'm not sure this line is legit

**Review by 0x72D0 — CHANGES_REQUESTED** [review]

File: docker-compose.local.yml (hunk 1 above)
Comment (Kam0ss) [reply docker-compose.local.yml]: You are right, this should not be here.

Comment (Kam0ss) [reply docker-compose.local.yml]: This has been fixed.

**Review by 0x72D0 — DISMISSED** [review]
LGTM!

**Review by 0x72D0 — APPROVED** [review]

**Pull request merged**